mix.exs

```diff
@@ -68,6 +69,7 @@ defmodule Swoosh.Mixfile do
       {:telemetry, "~> 0.4.2 or ~> 1.0"},
       {:hackney, "~> 1.9", optional: true},
       {:finch, "~> 0.6", optional: true},
+      {:req, "~> 0.4", optional: true},
```

Comment:
So let’s add `or ~> 1.0` here too, as it is coming out soon. :)

Reply:
Thanks!